conf/application.conf

```diff
@@ -106,4 +106,10 @@ assets {
   url = "http://localhost:9032/assets/"
 }
 
+urls {
```

Comment:
Do we need these links?
If these are Start page URLs, then we should probably treat them as content.

Reply:
ok, will move them to content

Reply:
done